src/sentry/api/endpoints/project_filter_details.py

```diff
@@ -68,3 +72,62 @@ def put(self, request, project, filter_id):
         )
 
         return Response(status=201)
+
+    # def put(self, request, project, filter_id):
```

Comment:
Is this a todo for the future?

Reply:
My bad ( shouldn't have been left there)